src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const personPartner = person.sex === 'm'
+    ? <p className="Person__partner">{`${person.partnerName} is my wife`}</p>
+    : <p className="Person__partner">Maksym is my husband</p>;
```

Comment:
Hm, _Maksym is my husband_ is hardcoded value and correct only for Olya.

You can use `person.partnerName` always and choose _wife_ or _husband_ text by this check: _person.sex === 'm'_